packages/esm-patient-registration-app/src/config-schema.ts

```diff
@@ -177,7 +177,24 @@ export const esmPatientRegistrationSchema = {
         _description: 'For coded questions only. Provide ability to add custom concept answers.',
       },
     },
-    _default: [],
+    _default: [
+      {
+        id: 'phone',
+        type: 'person attribute',
+        uuid: '14d4f066-15f5-102d-96e4-000c29c2a5d7',
+        validation: {
```

Comment:
Is this validation needed?

Reply:
No actually, the code broke as it wasn't passed and then I fixed the code line, but forgot to remove it from here.
Thanks for the catch!